lib/ssh/src/ssh_sftpd.erl

```diff
@@ -664,10 +664,11 @@ open(Vsn, ReqId, Data, State) when Vsn >= 4 ->
     do_open(ReqId, State, Path, Flags).
 
 do_open(ReqId, State0, Path, Flags) ->
-    #state{file_handler = FileMod, file_state = FS0, root = Root, xf = #ssh_xfer{vsn = Vsn}} = State0,
+    #state{file_handler = FileMod, file_state = FS0, xf = #ssh_xfer{vsn = Vsn}} = State0,
     XF = State0#state.xf,
     F = [binary | Flags],
```

Comment:
A matter of feeling, taste and reviewer...  I think that very small changes that increases the readability is OK.  Also remove dead code is OK.  But larger things, like re-structuring with other set of functions etc is not ok.  That is definitly for the master releases (20.0, 21.0 etc) since the risk for error introduction is not small.
But it is always good to first do the minimal fixes and get it running, and "cosmetic" stuff like this in separate commits at the end.